docs/en/sql-reference/aggregate-functions/reference/groupconcat.md

```diff
@@ -15,12 +15,19 @@ groupConcat[(delimiter [, limit])](expression);
 
 **Arguments**
 
-- `expression` — The expression or column name that outputs strings to be concatenated..
+- `delimiter` — A [string](../../../sql-reference/data-types/string.md) that will be used to separate concatenated values. This parameter is optional and defaults to an empty string or delimiter from parameters if not specified.
+- `expression` — The expression or column name that outputs strings to be concatenated.
+
+
+**Parameters**
+
 - `delimiter` — A [string](../../../sql-reference/data-types/string.md) that will be used to separate concatenated values. This parameter is optional and defaults to an empty string if not specified.
 - `limit` — A positive [integer](../../../sql-reference/data-types/int-uint.md) specifying the maximum number of elements to concatenate. If more elements are present, excess elements are ignored. This parameter is optional.
```

Comment:
If I've understood your point correctly, the info about `delimiter` was already stated in the parameters section because the delimiter could be used only as a parameter

Reply:
Oh, I wan't clear enough, sorry. I talked about `limit` field

Reply:
Oh, yes, `limit` was mentioned in the docs before: https://clickhouse.com/docs/en/sql-reference/aggregate-functions/reference/groupconcat
But it was marked as argument, and not parameter. From the syntax of the function in the docs, it is a parameter, and in the code it is a parameter as well, so I just clarified that those are parameters